src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -10153,3 +10153,570 @@ def test_aks_upgrade_upgrade_settings(self, resource_group, resource_group_locat
         # delete
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
+
+    @AllowLargeResponse()
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=88311518&view=results) to validate the change.

- test_aks_create_with_app_routing_enabled
- test_aks_approuting_enable_disable
- test_aks_approuting_enable_with_keyvault_secrets_provider_addon_and_keyvault_id
- test_aks_approuting_update
- test_aks_approuting_update_with_monitoring_addon_enabled
- test_aks_approuting_zone_add_delete_list
- test_aks_approuting_zone_update

Reply:
Created [new run with latest changes](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=88337759&view=results)